elements/signing/signed-kernel-endless.bst

```diff
@@ -0,0 +1,16 @@
+kind: eos_sb_signer
+description: |
+  Sign the Linux kernel binary using eos-sb-signer service.
+
+  See `doc/overview/signing.md` for context.
+
+build-depends:
+- components/linux.bst
+
+config:
+  # Path to the kernel binary in the dependency.
+  # The exact path may vary based on the kernel version and configuration.
+  input: "/boot/vmlinuz"
+
+  # Output path for the signed kernel.
+  output: "%{install-root}/boot/vmlinuz.signed"
```

Comment:
How does it send the kernel image to eos-sb-signer service for signing?  Or, the steps will show up in the following PR?

And, will the modules be signed, too?

Reply:
The kernel image is signed in `signing/signed-kernel-endless.bst`, which uses the `eos_sb_signer` element plugin.

The modules are signed as part of the `linux.bst` build process, using an ephemeral key. You can see this PR enables `MODULES_SIG_ALL` to make that happen. More info in the [Linux module signing docs](https://www.kernel.org/doc/html/latest/admin-guide/module-signing.html)